compiler/daml-lf-ast/src/DA/Daml/LF/Ast/Version.hs

```diff
@@ -85,11 +70,19 @@ featurePartyFromText = Feature "partyFromText function" version1_2
 supports :: Version -> Feature -> Bool
 supports version feature = version >= featureMinVersion feature
 
+renderMinorVersion :: MinorVersion -> String
```

Comment:
Should we make this return a `Text`?  Seems more in line with the usage sites.

Reply:
They are Text.Lazy. Having this return a lazy stream of text makes no sense. I'd suggest just sticking to String (but don't feel strongly about it).